GithubRepo/Scenes/About/AboutCoordinator.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AboutCoordinator.swift
+//  GithubRepo
+//
+//  Created by Marcos Alves on 18/10/21.
+//
+
+import Foundation
+import UIKit
+
+class AboutCoordinator: NavigationCoordinator {
+    var rootViewController: UINavigationController
+
+    var childCoordinators = [Coordinator]()
+
+    init(rootViewController: UINavigationController) {
+        self.rootViewController = rootViewController
+    }
+
+    func start() {
+        let aboutViewController = AboutViewController()
+        setup()
```

Comment:
You can explain better what the `setup` does. Firstly I thought that it was setup the `AboutCoordinator` but actually it is setup the root view controller

```suggestion
       self.setupRootViewController()
```